### PR TITLE
fix: search launch review

### PR DESCRIPTION
**Pull request by Xazin**
#### PR Checklist

- [ ] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

**Comment by annieappflowy** [comment]
I have not checked this new change. If you haven't added a hover tip on the search button, it would be better to one to inform the user of the search shortcut.

File: frontend/appflowy_flutter/lib/plugins/inline_actions/handlers/inline_page_reference.dart [hunk 1]
```diff
@@ -48,145 +47,100 @@ class InlinePageReferenceService extends InlineActionsDelegate {
...
+    // Filter by viewLayout
+    views.retainWhere(
+      (i) =>
+          currentViewId != i.id &&
+          (viewLayout == null || i.layout == viewLayout),
     );
 
-    _user = userProfile.fold((s) => s as UserProfilePB, (e) => null);
+    // Map to InlineActionsMenuItem, then take 5 items
+    return _recentViews = views.map(_fromView).take(5).toList();
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/inline_actions/handlers/inline_page_reference.dart:79]: shouldn't use `limitResults` here?

File: frontend/appflowy_flutter/lib/startup/startup.dart [hunk 2]
```diff
@@ -136,6 +138,7 @@ class FlowyRunner {
           if (isAppFlowyCloudEnabled) InitAppFlowyCloudTask(),
           const InitAppWidgetTask(),
           const InitPlatformServiceTask(),
+          getIt<CachedRecentService>(),
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/startup/startup.dart:141]: should not use it here. Create a launch task, wrap it, and dispose of the sources if needed.

**Comment by LucasXu0** [comment]
The cause of the failed tests is that the time for creating or removing a page has increased, so we have to increase the block response wait time for these tests. @Xazin, any changes to the creation or deletion of pages?

**Pull request merged**